deepspeech.go

```diff
@@ -105,7 +146,16 @@ func (s *Stream) IntermediateDecode() string {
 // FinishStream Signal the end of an audio signal to an ongoing streaming
 // inference, returns the STT result over the whole audio signal.
 func (s *Stream) FinishStream() string {
-	return C.GoString(C.FinishStream(s.sw))
+	str := C.FinishStream(s.sw)
+	retval := C.GoString(str)
+	defer C.FreeString(str)
```

Comment:
You should move the `defer` one line above

Reply:
Done